app/components/Settings/Settings.react.js

```diff
@@ -62,7 +64,9 @@ class Settings extends Component {
 
     componentDidMount() {
         this.fetchData();
-        this.props.dispatch(Actions.hasCerts());
+        checkConnectorSettings = setInterval(() => {
+            this.props.dispatch(Actions.getConnectorSettings());
+        }, 2000);
```

Comment:
The real certificates take a minute to receive - connector can't know the final domain (hash generated by bot).